### ISSUE TITLE
Why you don't have batchnorm for first layer of critic?

### ISSUE THREAD

**Issue by menglin0320** [Zardinality/WGAN-tensorflow#1]
Some sepcial reason?

**Comment by kang205** [comment]
It follows the same setting in DCGAN and PyTorch version of PyTorch.

**Comment by Zardinality** [comment]
Yes. I just follows the model defined in PyTorch version of WGAN. No special reason for this. But since even MLP can do a decent job as a critic, you might want to try how it works with bn in the first layer.

**Comment by songhaohaohao** [comment]
Hello, I'm sorry to bother you, but I have some question, the mnist can be replaced with a pair of image?and  can it be extracted to  haracteristics ?

**Comment by songhaohaohao** [comment]
Hello, I'm sorry to bother you, but I have some question, the mnist can be replaced with a pair of image?and can it be extracted to characteristics ?
